README.md

```diff
@@ -44,9 +44,12 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 | message              | messedż             | widomość                              |
 | mockup               | mokap               | makieta                               |
 | paging               | paginacja           | stronicowanie                         |
+| Proof of Concept/PoC | piołsi              | dowód słuszności (przyjętych założeń) |
 | process              | procesować          | przetwarzać                           |
 | progress             |                     | postęp                                |
 | property             |                     | właściwość, opcja, cecha              |
+| prototype            | prototajp           | prototyp                              |
+| release              | rilis               | wydanie, wersja                       |
```

Comment:
Doszedłem do wniosku, żeby unikać prostego zapisu fonetycznego (https://github.com/nurkiewicz/polski-w-it/blob/master/CONTRIBUTING.md), możesz usunąć?